hs_file_types/tests/test_view_functions.py

```diff
@@ -0,0 +1,220 @@
+import os
+import json
+import tempfile
+import shutil
+
+from django.core.files.uploadedfile import UploadedFile
+from django.test import TestCase, RequestFactory
+from django.contrib.auth.models import Group
+from django.core.urlresolvers import reverse
+
+from rest_framework import status
+
+from hs_core import hydroshare
+from hs_core.hydroshare.utils import resource_post_create_actions
+from hs_core.testing import MockIRODSTestCaseMixin
+from ..views import set_file_type, add_metadata_element, update_metadata_element, \
+    update_key_value_metadata, delete_key_value_metadata
+from ..models import GeoRasterLogicalFile
```

Comment:
@pkdash better to use absolute imports rather than relative imports. Here is recommendation from PEP 8:
```
Relative imports for intra-package imports are highly discouraged. Always use the absolute package path for all imports. Even now that PEP 328 [7] is fully implemented in Python 2.5, its style of explicit relative imports is actively discouraged; absolute imports are more portable and usually more readable.
```